StortSpelProjekt/Sandbox/imgui.ini

```diff
@@ -1,15 +0,0 @@
-[Window][Debug##Default]
-Pos=60,60
-Size=400,400
-Collapsed=0
-
-[Window][Debugging info]
-Pos=640,0
-Size=640,51
-Collapsed=0
-
-[Window][Console]
-Pos=0,0
-Size=640,720
-Collapsed=1
-
```

Comment:
Kill it before it lays eggs

Reply:
Har lagt till den i gitignore för hundrafjortonde gången